lib/liveValidation/liveValidator.ts

```diff
@@ -993,21 +966,18 @@ export function formatUrlToExpectedFormat(requestUrl: string): string {
 /**
  * Parse the validation request information.
  *
- * @param  requestUrl The url of service api call.
+ * @param requestUrl The url of service api call.
  *
  * @param requestMethod The http verb for the method to be used for lookup.
  *
- * @param correlationId The id to correlate the api calls.
- *
- * @param activityId The id maps to request id, used by RPaaS.
+ * @param headers Optional headers
  *
  * @returns parsed ValidationRequest info.
  */
 export const parseValidationRequest = (
   requestUrl: string,
-  requestMethod: string | undefined | null,
-  correlationId: string,
-  activityId: string
+  requestMethod: string,
```

Comment:
Can you revert type change which will break current usage?

Reply:
okay, I will restore the legacy one. After the usage in live validation service is updated, we can remove it then.

Reply:
fixed, the exported name in index.ts is kept same as before.